.azure-pipelines/client.test.live.yml

```diff
@@ -311,3 +310,590 @@ jobs:
         inputs:
           artifactName: uamqp-linux-$(PythonVersion)-whl
           pathToPublish: 'dist'
+
+  - job: 'EventHub_LiveTest_windows2019_27'
+    timeoutInMinutes: 120
+    dependsOn: 'Windows'
+    pool:
+      name: 'azsdk-pool-mms-win-2019-general'
+      vmImage: 'MMS2019'
+    strategy:
+      maxParallel: 1
+      matrix:
+        x64 Python 2.7:
+          PythonArchitecture: 'x64'
+          PythonVersion: '$(PythonVersion27)'
+          GetPip: 'https://bootstrap.pypa.io/pip/2.7/get-pip.py'
+    variables:
+      DevReqsPath: './azure-sdk-for-python/sdk/eventhub/azure-eventhub/dev_requirements.txt'
+      DownloadArtifactFolder: 'uamqp-win$(PythonArchitecture)-$(PythonVersion)-whl'
+      SdkPath: 'azure-sdk-for-python/sdk/eventhub/azure-eventhub'
+    steps:
+      - template: /.azure-pipelines/use-python-version.yml
+        parameters:
+          architecture: '$(PythonArchitecture)'
+          versionSpec: '$(PythonVersion)'
+      - script: |
+          git clone https://github.com/Azure/azure-sdk-for-python.git --single-branch
+        displayName: 'Clone azure-sdk-for-python'
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Downloading artifact'
+        inputs:
+          source: 'current'
+          patterns: '$(DownloadArtifactFolder)/*.whl'
+      - pwsh: |
+          $whlfile = Get-ChildItem -Filter *.whl $(Pipeline.Workspace)/$(DownloadArtifactFolder) | Select-Object -First 1 -ExpandProperty Name
+          echo "##vso[task.setvariable variable=DownloadArtifactWhl]$whlfile"
+          dir $(Pipeline.Workspace)/$(DownloadArtifactFolder)
```

Comment:
When you're running `pwsh`, `dir` is just aliased `Get-ChildItem`.

![image](https://user-images.githubusercontent.com/45376673/117499914-c3035a80-af30-11eb-8457-c75dad70df6a.png)